src/KM_Controls.pas

```diff
@@ -6427,8 +6709,21 @@ procedure TKMMasterControl.UpdateFocus(aSender: TKMControl);
 end;
 
 
+//Check if control is covered by other controls or not
+//We assume that control is covered if any of his 4 angles are covered
+//Use Self coordinates to check, because some controls can contain other controls (f.e. TKMNumericEdit)
+function TKMMasterControl.IsCtrlCovered(aCtrl: TKMControl): Boolean;
+begin
+  Result := (HitControl(aCtrl.SelfAbsLeft, aCtrl.SelfAbsTop) = aCtrl)
+        and (HitControl(aCtrl.SelfAbsLeft + aCtrl.SelfWidth - 1, aCtrl.SelfAbsTop) = aCtrl)
+        and (HitControl(aCtrl.SelfAbsLeft, aCtrl.SelfAbsTop + aCtrl.SelfHeight - 1) = aCtrl)
+        and (HitControl(aCtrl.SelfAbsLeft + aCtrl.SelfWidth - 1, aCtrl.SelfAbsTop + aCtrl.SelfHeight - 1) = aCtrl);
+  Result := not Result;

```

Comment:
It would be less confusing to switch "=" with "<>" and "and" with "or" so you don't need to do Result := not Result